mslib/ogcwms.py

```diff
@@ -621,3 +641,9 @@ def readString(self, st):
         if not isinstance(st, str):
             raise ValueError("String must be of type string, not %s" % type(st))
         return etree.fromstring(st)
+
+
+def removeXMLNamespace(tree):
+    for elem in tree.iter():
+        if elem.tag.startswith("{"):
+            elem.tag = elem.tag.split("}")[-1]
```

Comment:
I ran into an issue when testing a third party OGC WMS Server with MSS.
Namely when requesting their GetCapabilities, the elements weren't just called "Capabilities" but "{Namespace}Capabilities" causing MSS to fail finding the element.
By calling this function, the {Namespace} part is first removed before searching in it.
Is this a good or a bad thing?

Reply:
That seems strange. Is this standard behaviour? What third party server was that?
Those that I tested for 1.1.1 didn't behave in that fashion (I discovered many problems in MSS, but this seems an unreasonable answer with {} in the reply?

Reply:
I seem to have missed that part, but yes, namespaces are now required per the standard in 1.3.0.
This is something I still have to add for the server then.

Reply:
The MSS Server now also contains namespaces for 1.3.0.
It is indeed required and not optional, I will quote from the 1.3.0 standard for getcapabilities:

> This  element  shall  contain   an   XML   Schema   instance   schemaLocation   attribute   that   binds   the   “http://www.opengis.net/wms” namespace  to  the  schema  in  E.1. 

I also went ahead and did the same for exceptions. While that isn't required per the document, their examples and other Servers still do that.